metadata.yaml

```diff
@@ -1,5 +1,5 @@
 name: alertmanager-k8s
-display-name: Alertmanager
+display-name: Alertmanager K8S
```

Comment:
Please no K8S in the display name, qualifying something for K8S is something we do only when necessary

Reply:
Agree.

Reply:
I was following an (semi?) established pattern: currently in charmhub there is "K8S" in alertmanager, prometheus, grafana, cassandra.

Reply:
Charmhub has the following alertmanager machine charms, so keep the "K8S" for this one?
https://charmhub.io/charmcraft-alertmanager
https://charmhub.io/prometheus-alertmanager